dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -3995,7 +3995,7 @@ public void copyProperties(final Contentlet contentlet, final Map<String, Object
                     }else if(isFieldTypeDate(field)){
                         contentlet.setDateProperty(conVariable,value != null ? (Date)value : null);
                     }else if(isFieldTypeLong(field)){
-                        contentlet.setLongProperty(conVariable,value != null ? (Long)value : null);
+                        contentlet.setLongProperty(conVariable,value != null ? ((Number)value).longValue(): null);
```

Comment:
ConversionUtils does a better job on it

Reply:
I think it would do a better job. if we had an object of unknown type. Cuz ConversionUtils handles o CharSequence or a number (We know for sure this is not a CharSequence) it's a number which is already handled here. The NumberFormatException will never be thrown here. 
Additionally, the default Long is only assigned when the number format exception arises.

if we really want to refactor this whole copyLogic it should be handled on a separate task.